benchmarks/src/bin/h2o.rs

```diff
@@ -113,13 +118,16 @@ async fn group_by(opt: &GroupBy) -> Result<()> {
     let start = Instant::now();
     let df = ctx.sql(sql).await?;
     let batches = df.collect().await?;
-    let elapsed = start.elapsed().as_millis();
-
+    let elapsed = start.elapsed().as_secs_f64() * 1000.0;
+    let numrows = batches.iter().map(|b| b.num_rows()).sum::<usize>();
     if opt.debug {
         pretty::print_batches(&batches)?;
     }
-
+    rundata.write_iter(elapsed, numrows);
     println!("h2o groupby query {} took {} ms", opt.query, elapsed);
 
+    if let Some(path) = &opt.output_path {
```

Comment:
minor: maybe we could encapsulate the write here inside `rundata`:

```rust
  run_data.maybe_write_json(opt.output_path.as_ref())?
```

Reply:
yeah this might be nicer.